agreementbot/secure_api.go

```diff
@@ -189,6 +192,7 @@ func (a *SecureAPI) listen() {
 		router.HandleFunc(`/org/{org}/secrets/user/{user}/{secret:[\w\/\-]+}`, a.userSecret).Methods("GET", "LIST", "PUT", "POST", "DELETE", "OPTIONS")
 		router.HandleFunc("/org/{org}/secrets", a.orgSecrets).Methods("LIST", "OPTIONS")
 		router.HandleFunc(`/org/{org}/secrets/{secret:[\w\/\-]+}`, a.orgSecret).Methods("GET", "LIST", "PUT", "POST", "DELETE", "OPTIONS")
+		router.HandleFunc("/org/{org}/hanode/{group}/{node}/{nmpid}", a.haNodeNMPUpdateRequest).Methods("POST", "OPTIONS")
```

Comment:
Please add the word 'nodemanagement' in the path so that the caller understands what it is for, though it is used internally by the agent. Like `/nodemanagement/hagroup/{org}/{group}/{node}/{nmpid}`